python/cuml/testing/test_preproc_utils.py

```diff
@@ -31,7 +31,7 @@
 from cuml.common import input_to_cuml_array
 
 
-def to_output_type(array, output_type, order='F'):
+def to_output_type(array, output_type, order='K'):
```

Comment:
Any particular reason the default was changed here? I would prefer `K` myself, but have we checked that we don't depend on a default of `F` anywhere either for correctness or performance?

Reply:
Used that while debugging, I just reversed it.